test-suite/src/lib.rs

```diff
@@ -14,6 +17,7 @@ pub async fn run_command(command: impl AsRef<str>) {
 
 #[cfg(not(target_os = "windows"))]
 pub async fn run_command(command: impl AsRef<str>) {
+    println!("> RUN: {}", command.as_ref());
```

Comment:
```suggestion
```

Reply:
why?

Reply:
I thought it was just for you running the test locally

Reply:
junha's reply: it's a feature to show which command it was for the following results of the command